cmd/explaintest/r/tpch.result

```diff
@@ -124,7 +124,7 @@ Sort	2.94	root		tpch.lineitem.l_returnflag, tpch.lineitem.l_linestatus
   └─HashAgg	2.94	root		group by:tpch.lineitem.l_linestatus, tpch.lineitem.l_returnflag, funcs:sum(Column#26)->Column#18, funcs:sum(Column#27)->Column#19, funcs:sum(Column#28)->Column#20, funcs:sum(Column#29)->Column#21, funcs:avg(Column#30, Column#31)->Column#22, funcs:avg(Column#32, Column#33)->Column#23, funcs:avg(Column#34, Column#35)->Column#24, funcs:count(Column#36)->Column#25, funcs:firstrow(tpch.lineitem.l_returnflag)->tpch.lineitem.l_returnflag, funcs:firstrow(tpch.lineitem.l_linestatus)->tpch.lineitem.l_linestatus
     └─TableReader	2.94	root		data:HashAgg
       └─HashAgg	2.94	cop[tikv]		group by:tpch.lineitem.l_linestatus, tpch.lineitem.l_returnflag, funcs:sum(tpch.lineitem.l_quantity)->Column#26, funcs:sum(tpch.lineitem.l_extendedprice)->Column#27, funcs:sum(mul(tpch.lineitem.l_extendedprice, minus(1, tpch.lineitem.l_discount)))->Column#28, funcs:sum(mul(mul(tpch.lineitem.l_extendedprice, minus(1, tpch.lineitem.l_discount)), plus(1, tpch.lineitem.l_tax)))->Column#29, funcs:count(tpch.lineitem.l_quantity)->Column#30, funcs:sum(tpch.lineitem.l_quantity)->Column#31, funcs:count(tpch.lineitem.l_extendedprice)->Column#32, funcs:sum(tpch.lineitem.l_extendedprice)->Column#33, funcs:count(tpch.lineitem.l_discount)->Column#34, funcs:sum(tpch.lineitem.l_discount)->Column#35, funcs:count(1)->Column#36
-        └─Selection	293795345.00	cop[tikv]		le(tpch.lineitem.l_shipdate, 1998-08-15)
+        └─Selection	293795345.00	cop[tikv]		le(tpch.lineitem.l_shipdate, 1998-08-15 00:00:00.000000)
```

Comment:
before this pr:
    l_shipdate <= date_sub('1998-12-01', interval 108 day) is datetime < datetime( 1998-09-18), and it will be 
    explained to le(tpch.lineitem.l_shipdate, 1998-08-15)
after this pr:
   l_shipdate <= date_sub('1998-12-01', interval 108 day) is datetime < string( "1998-09-18"), and string("1998-09-18") will be converted to datetime(1998-08-15 00:00:00.000000), so that this plan will be explained to le(tpch.lineitem.l_shipdate, 1998-08-15 00:00:00.000000)